### DISCUSSION TITLE
border-y and border-x

### DISCUSSION THREAD

**Discussion by badosz0** [tailwindlabs/tailwindcss#4303] [Ideas]
Tailwind is missing border-x and border-y and it could be very useful in my opinion (I wanted to use it today, just like mx, px etc. and discovered that it doesn't exist.) In #182 one of the reasons of border-y and border-x not existing is generating a lot of new classes and I think it's not a big problem anymore with JIT-mode?

**Comment by simonswiss** [comment]
That's certainly less of an issue with JIT. Worth noting JIT is still an "opt-in" option, and not the default at this point, so perhaps at the moment those `border-x` and `border-y` utilities still belong to a plugin or `@utilities` layer in the CSS.

**Comment by DariosKrimsKrams** [comment]
I agree with @badosz0, `border-x` and `border-y` would be nice :)
